lessons/programming/algorithm/index.mdx

```diff
@@ -1,76 +1,83 @@
 ---
-title: Algorithm
+title: Algorithms
 ---
 
-# Algorithm
+# Algorithms
 
 ### An algorithm is the set of instructions in steps to solve a problem.
 
 In real-life if you had to count the number of members in a room, you would count one by one until you finish counting the last member.
 
-So we count a member and add one to the previous counted number, go to next member, and repeat the same procedure until there are no members in the room. This is a step-wise procedure and that a normal person follows to count the members.
+So we count a member and add one to the previous counted number, go to next member, and repeat the same procedure until there are no members in the room. This is a step-wise procedure we use to count the number of members.
 
-For a computer to solve a problem or perform a task for us, for example, a counting poblem, we give it a sequence of instructions similar to the step-bt-step procedure humans follow ourselves. An algorithm is a sequence of instructions given to computers to solve problems or perform tasks.
+For a computer to solve a problem or perform a task for us, for example, a counting poblem, we give it a sequence of instructions similar to the step-bt-step procedure humans follow. An algorithm is a sequence of instructions given to computers to solve problems or perform tasks.
 
-An algorithm can be written by a plain langugae, a pseudo-code or described by a diagram-representd flowchart and it can be implemented in different computer programming langugages. 
+An algorithm can be written by a plain langugae, a pseudo-code or described by a diagram-based flowchart and it can be implemented in different computer programming langugages. 
```

Comment:
An algorithm can be written ~~by~~ -> **in** a plain ~~langugae~~ -> **language**